config/vulnDictConf.go

```diff
@@ -328,3 +328,9 @@ func (cnf *CtiConf) Init() {
 	cnf.setDefault("go-cti.sqlite3")
 	cnf.DebugSQL = Conf.DebugSQL
 }
+
+type Vuls2DictConf struct {
```

Comment:
```suggestion
// Vuls2DictConf is vuls2 config
type Vuls2DictConf struct {
```

Reply:
done in https://github.com/future-architect/vuls/compare/e75c2d1ced0770bb88e2b8a104eed7a055ae3ff0..894b30d925029a2d671fad686dd87ea806e29cf9